lib/Redmine/Api/Membership.php

```diff
@@ -101,6 +101,30 @@ public function remove($id)
         return $this->delete('/memberships/'.$id.'.xml');
     }
 
+    /**
+     * Delete membership of project by user id
+     *
+     * @param int $project_id id of project
+     * @param int $user_id id of user
+     *
+     * @return false|\SimpleXMLElement|string
+     */
+    public function removeMember($project_id, $user_id)
+    {
+        $memberships = $this->all($project_id);
+
+        if (isset($memberships['memberships']))
```

Comment:
try running php-cs-fixer on this file please, it should fix the cs errors
thanks!

Reply:
I have fixed the camel case vars.  I do not know how to use php-cs-fixer and don't have time to figure it out, sorry.

Reply:
OK, changes have been made.